app/views.py

```diff
@@ -1,131 +1,134 @@
-@app.route('/get_section_articles/sections/<string:section_slug>/subsection/<string:subsection_slug>/articles/<string:article_slug>', methods=['GET']) 
-def articles_in_section(section_slug,subsection_slug,article_slug):
-    if article_slug != None and article_slug != "None":
-        articles = [models.Article.query.filter(
-            models.Article.slug == article_slug
-                ).first()]
-    elif subsection_slug == "main":
-        section =  models.Section.query.filter(
-            models.Section.slug == section_slug
-                ).first()
-        articles = models.Article.query.filter(
-            models.Article.section == section
-                ).all()
-    else:
-        subsection =  models.Subsection.query.filter(
-            models.Subsection.slug == subsection_slug
-                ).first()
-        articles = models.Article.query.filter(
-            models.Article.subsection == subsection
-                ).all()
-    secure_articles = []
-    for article in articles:
-        article_dict = {
-      "content": article.content, 
-      "datetime": article.datetime, 
-      "id": article.id, 
-      "isDraft": article.isDraft, 
-      "issue": article.issue, 
-      "section_id": article.section_id, 
-      "slug": article.slug, 
-      "subsection_id": article.subsection_id, 
-      "title": article.title, 
-      "volume": article.volume
-                    }   
-        secure_articles.append(article_dict)
-    return jsonify({"articles": secure_articles})
 
+#----------------------------------------------  
 
-#----------------------------------------------    
-
-@app.route('/newspaper/<int:volume>/<int:issue>', methods=['GET'])
+@app.route('/newspaper/<int:volume>/<int:issue>' )
 def retrieve_article_data(volume,issue):
-    articles = models.Article.query.filter(models.Article.volume == volume 
-        and models.Article.issue == issue).all()
-    secure_articles = []
-    for article in articles:
-        article_dict = {
-      "content": article.content, 
-      "datetime": article.datetime, 
-      "id": article.id, 
-      "isDraft": article.isDraft, 
-      "issue": article.issue, 
-      "section_id": article.section_id, 
-      "slug": article.slug, 
-      "subsection_id": article.subsection_id, 
-      "title": article.title, 
-      "volume": article.volume
-                    }   
-        secure_articles.append(article_dict)
-    issuu_code = models.Issuu.query.filter(models.Issuu.volume == volume 
-        and models.Issuu.issue == issue).first().code
-    return jsonify({"issuu_code": issuu_code, "articles": secure_articles})
+  articles = models.Article.query.filter(models.Article.volume == volume 
+    and models.Article.issue == issue).all()
+  secure_articles = []
+  for article in articles:
+    article_dict = {
+    "content": article.content, 
+    "datetime": article.datetime, 
+    "id": article.id, 
+    "isDraft": article.isDraft, 
```

Comment:
you changed this to snakecase in the model. should be "is_draft". this code is obviously not tested.